assets/javascripts/discourse/helpers/activity-pub-actor-handle.js

```diff
@@ -0,0 +1,16 @@
+import { htmlSafe } from "@ember/template";
+import { registerRawHelper } from "discourse-common/lib/helpers";
+import { buildHandle } from "../lib/activity-pub-utilities";
+
+function renderActivityPubActorHandle(actor) {
+  if (actor) {
+    const handle = buildHandle({ actor });
+    return `<a href='${actor.url}' target="_blank" rel="noopener noreferrer">${handle}</a>`;
+  }
+}
+
+registerRawHelper("activityPubActorHandle", activityPubActorHandle);
```

Comment:
In core, we're moving away from helpers. Can we try and do this without a helper?

Reply:
Done: https://github.com/angusmcleod/discourse-activity-pub/commit/3d329999a75948030095e28f10ea5233161cbe33